platform/Stream.cpp

```diff
@@ -112,14 +103,14 @@ void Stream::rewind() {
 }
 
 int Stream::isatty() {
-    return 0;
+    return 1;
```

Comment:
I think I agree with geky - leave it untouched, as we're planning to deprecate. The Stream is always self-opening, so can set unbuffered itself regardless of the wrong isatty, as it always has. We don't really want to be breaking any Serial-using code (as the fdopen change itself did, until we spotted the need to change isatty).

When to put the deprecation markers in? I think we're deprecating Stream, and hence Serial, leaving RawSerial, FileHandle and BufferedSerial.